packages/pages/src/App.js

```diff
@@ -19,30 +19,35 @@ const App = ({ effects, state }) => (
             >
               Delete
             </button>
+            <button
+              type="button"
+              data-id={entry.id}
+              data-name={entry.name}
+              data-content={entry.content}
+              onClick={effects.updateEntry}
+            >
+              Update
+            </button>
           </li>
         ))}
       </ul>
-    </div>
-    <div name="createEntry">
-      <label>
-        Name
-        <input
-          type="text"
-          value={state.name}
-          onChange={effects.changeName}
-        />
-      </label>
-      <label>
-        Content
-        <input
-          type="text"
-          value={state.content}
-          onChange={effects.changeContent}
-        />
-      </label>
-      <button type="button" onClick={effects.createEntry}>
-        Create
-      </button>
+      <form onSubmit={effects.submit}>
+        <label>
+          Name
+          <input type="text" value={state.name} onChange={effects.changeName} />
+        </label>
+        <label>
+          Content
+          <input
+            type="text"
+            value={state.content}
+            onChange={effects.changeContent}
+          />
+        </label>
+        <button type="submit" value="Submit">
+          {state.id !== "" ? "Update" : "Create"}
+        </button>
+      </form>
```

Comment:
You should add a *Reset* button to:
- empty the fields
- cancel the edition if any